customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.plulist.v1.json",
+  "title": "PLU List messages",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "id",
+      "plu"
+    ],
+    "properties": {
+      "effectiveDate": {
+        "description": "The date when this PLU list should be applied, if present.",
+        "type": "string",
+        "format": "date",
+        "example": "2021-01-01"
+      },
+      "id": {
+        "description": "A unique identifier for the PLU list.",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 20,
+        "example": "top-5-items-plu"
+      },
+      "name": {
+        "description": "The PLU List name",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 40,
+        "example": "PLU List"
+      },
+      "description": {
+        "description": "The PLU List description",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 255,
+        "example": "The description for PLU List"
+      },
+      "deviceTypes": {
+        "description": "A list of device types that this PLUList applies to",
+        "type": "array",
+        "example": "[{'id': 'SelfCheckout'}]",
```

Comment:
lets define fixed list of such ids and try to stick to it
on our end we have now 4 types:

PLU_BAGS_SCO
PLU_MPOS
PLU_SCO
PLU_POS

if that's acceptable, let's keep them that way?

Reply:
Added those strings as examples. Is that enough or should we use an enum instead?

Reply:
i'd use a string, to simplify future addition of new types. if we go with enums, we will have to update scheme here every time